uni/android/app/build.gradle

```diff
@@ -52,7 +52,7 @@ android {
         applicationId "pt.up.fe.ni.uni"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-build-configuration.
-        minSdkVersion 19
+        minSdkVersion 21 // default is flutter.minSdkVersion
```

Comment:
Bumping the minSdkVersion to avoid problems with [multidex support](https://docs.flutter.dev/deployment/android#enabling-multidex-support), which this PR triggered due to the addition of a lot of new methods